test/unit/components/compose-comment.test.jsx

```diff
@@ -292,19 +301,15 @@ describe('Compose Comment test', () => {
                 }
             }
         });
-        const component = mountWithIntl(
-            <ComposeComment
-                {...defaultProps()}
-            />
-            , {context: {store}}
-        );
+        const {instance, findByComponentName} = getComposeCommentWrapper({}, store);
         // set state on the ComposeComment component, not the wrapper
-        const commentInstance = component.find('ComposeComment').instance();
-        commentInstance.setState({muteOpen: true});
-        component.update();
-        expect(component.find('MuteModal').exists()).toEqual(true);
-        expect(component.find('MuteModal').props().startStep).toEqual(0);
-        expect(component.find('MuteModal').props().showWarning).toBe(false);
+        const commentInstance = instance();
+        act(() => {
+            commentInstance.setState({muteOpen: true});
+        });
+        expect(screen.getByRole('dialog')).toBeInTheDocument();
+        expect(findByComponentName('MuteModal').props.startStep).toEqual(0);
+        expect(findByComponentName('MuteModal').props.showWarning).toBe(false);
```

Comment:
Same questions as the other comment. I see several instances further down, too. Just checking about consistency. If there's a reason to do it this way, I don't object :)